cmd/mutex.go

```diff
@@ -77,10 +104,12 @@ var getSubCmd = &cobra.Command{
 
 // initialization of variables
 func init() {
+	mutexCmd.PersistentFlags().BoolVar(&hasTLS, "tlsauth", false, "Use mTLS auth to connect to etcd, if this is set, --etcdpass and --etcduser are ignored. Cert info will be read from config/ GROMIT_ETCD_CA, GROMIT_ETCD_CLIENT_CERT, GROMIT_ETCD_CLIENT_KEY must be set")
 	mutexCmd.PersistentFlags().StringVar(&etcdPass, "etcdpass", os.Getenv("ETCD_PASS"), "Password for etcd user")
 	mutexCmd.PersistentFlags().StringVar(&etcdUser, "etcduser", "root", "etcd user to connect as")
-	mutexCmd.PersistentFlags().StringVar(&etcdHost, "host", "ec2-3-66-86-193.eu-central-1.compute.amazonaws.com:2379", "etcd host")
+	mutexCmd.PersistentFlags().StringVar(&etcdHost, "host", "", "etcd host")
 	mutexCmd.PersistentFlags().StringVar(&script, "script", "testdata/mutex/script.sh", "script to be run after acquiring lock")
+	mutexCmd.MarkFlagsMutuallyExclusive("tlsauth", "etcduser")
```

Comment:
Even if someone accidentally/unknowingly sets tlsauth along with user auth - it will have etcduser and etcdpass. etcdpass alone doesn't make sense. So figured user alone should be alright, as tlsauth and etcduser can't be set together.